thunder/dynamo/report.py

```diff
@@ -607,6 +615,11 @@ def __init__(self, bsym: BoundSymbol, name: str):
     def __repr__(self):
         return f"<ThunderFusionReport of bound symbol\n{self.nvfusion_bsym}>"
 
+    def benchmark(self, fn: Callable, timer_fn: Callable):
+        compiled_fn = fn(self.nvfusion_bsym)
+        inputs = self.get_inputs()
+        return timer_fn("compiled_fn(*inputs)", globals={"compiled_fn": compiled_fn, "inputs": inputs})
```

Comment:
Interesting that the timer takes a string to execute and not a Python callable -- I'm guessing that's just the typical interface for timers?